modules/api/cmd/kubermatic-api/swagger.json

```diff
@@ -12,7 +12,7 @@
   "info": {
     "description": "This OpenAPI 2.0 specification describes the REST APIs used by the Kubermatic Kubernetes Platform Dashboard.",
     "title": "Kubermatic Kubernetes Platform API",
-    "version": "2.23"
+    "version": "2.26"
```

Comment:
`swagger.json` is generated, we can't change it manually. The source is this line: https://github.com/kubermatic/dashboard/blob/main/modules/api/cmd/kubermatic-api/main.go#L22.

Reply:
Bit confused since I reference this [commit ](https://github.com/kubermatic/dashboard/commit/b6d97439f61eae5218f0bf42e8a09af51327922c) now and there is manual change to swagger.json. So should I keep it or revert if there is any change in process later.

Reply:
You can keep it I suppose, the `swagger.json` _should_ be changed, but it should be because you ran codegen scripts, not because you updated it yourself.